resources/freedesktop-sdk.ld.so.blockedlist

```diff
@@ -1,82 +1,82 @@
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libsoftokn3.so
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libwayland-client.so.0
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libwayland-server.so.0
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libX11-xcb.so.1
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libXau.so.6
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libxcb-dri2.so.0
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libxcb-glx.so.0
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libxcb-xfixes.so.0
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libXcomposite.so.1
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libXcursor.so.1
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libXdamage.so.1
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libXdmcp.so.6
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libXext.so.6
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libXfixes.so.3
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libXi.so.6
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libxml2.so.2
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libXrender.so.1
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libxslt.so.1
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libXtst.so.6
+*(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/libexec/QtWebEngineProcess *(*/)Sid\ Meier\'s\ Civilization\ VI/GameGuide/lib/libXxf86vm.so.1
+*(*/)Steam/steamapps/common/*(*/)* *(*/)steam-runtime/+(*/)libSDL2-2.0.so.0.9.0
```

Comment:
If you install a game in a different "content library folder", then "Steam" prefix is not there. You should match "*(/*)/steamaps/common/*(*/)*"

Reply:
There are some * that were removed by Mardown.

Reply:
Right, that makes sense. I'll anyway move to use YAML config before continuing with this.